scripts/test-e2e-local-clean.js

```diff
@@ -44,24 +44,25 @@ if (isPackagerRunning() === 'running') {
 console.info('\n** Cleaning Gradle build artifacts **\n');
 exec('./gradlew clean');
 exec('rm -rf /tmp/maven-local');
+exec('rm -rf /tmp/maven-local.zip');
+exec('rm -rf /tmp/rntester.apk');
```

Comment:
instead of putting many different files across the `/tmp/*` folder, can we create a `tmp/react-native-temp/*` folder and put all the various things inside it? it'd be cleaner and it will mean that here in the remove script we will need to only remove the `/tmp/react-native-temp/` folder.